annotation-processor/src/test/java/org/hibernate/validator/ap/testutil/CompilerTestHelper.java

```diff
@@ -180,16 +181,16 @@ public boolean compile(Processor annotationProcessor,
 		List<String> options = new ArrayList<String>();
 
 		if ( diagnosticKind != null ) {
-			options.add( String.format( "-A%s=%s", Configuration.DIAGNOSTIC_KIND_PROCESSOR_OPTION, diagnosticKind ) );
+			options.add( StringHelper.format( "-A%s=%s", Configuration.DIAGNOSTIC_KIND_PROCESSOR_OPTION, diagnosticKind ) );
```

Comment:
Just thinking out loud.. Is it OK that we are "leaking" internal util into another module? And I've checked, we have the `StringHelper` in AP which is almost a copy of the one from the engine. I'd probably would either keep the changes to the engine only, or add that same method to AP helper. But I'll leave it for you to decide :)